ecommerce/CouponController.php

```diff
@@ -456,14 +456,42 @@ public function tabs_key_value(): array {
 		);
 
 		foreach ( $coupon_status as $key => $value ) {
-			$where['coupon_status'] = $key;
+			if ( CouponModel::STATUS_EXPIRED === $key ) {
+				$where['coupon_status'] = CouponModel::STATUS_ACTIVE;
```

Comment:
Do not convert to UNIX_TIMESTAMPT. It make slower query and has side effect based on server configuration.
```
$gmt_datetime = DateTimeHelper::now()->to_date_time_string();
```
Now pass it to 
```

Change the others query accordingly
$raw_query = 'start_date_gmt > %s OR expire_date_gmt < %s';
$where[ $raw_query ]    = array(
					'RAW',
					array( $gmt_datetime, $gmt_datetime ),
				);
```